proto/penumbra/penumbra/view/v1/view.proto

```diff
@@ -170,6 +174,16 @@ message AuctionsResponse {
   // Only present when `query_latest_state` was provided.
   // Could be empty, depending on the auction state.
   repeated core.component.dex.v1.Position positions = 3;
+  // The sequence number of the auction state _as known to the local view
+  // service_. Note that the local view service may lag behind the fullnode. For
+  // example, if the chain hits an auction's `end_height`, but the user hasn't
+  // yet exchanged their sequence-0 (opened) auction NFT for a sequence-1
+  // (closed) auction NFT, the local view service will have a sequnce number of
+  // 0.
+  //
+  // Dutch auctions move from:
+  // 0 (opened) => 1 (closed) => n (withdrawn)
+  uint64 local_seq = 5;
```

Comment:
`note_record` above uses `4`, which is why this field uses `5`

Reply:
oh strange, could you move it so that they're ordered 🙏

Reply:
done